src-vue/lib/SSH.ts

```diff
@@ -12,13 +12,13 @@ export interface ITryServerData {
 }
 
 export class SSH {
-  private static connection: SSHConnection;
+  private static connection?: SSHConnection;
   private static config: Config;
 
   public static setConfig(config: Config): void {
     this.config = config;
     if (this.connection) {
-      this.reconnect();
+      void this.reconnect();
```

Comment:
Once the config has been switched, I don't believe a reconnect is the right method. It should create a new connection with the new config. I can fix after the merge.